doc/source/whatsnew/v1.6.0.rst

```diff
@@ -33,6 +33,7 @@ Other enhancements
 - :meth:`Series.add_suffix`, :meth:`DataFrame.add_suffix`, :meth:`Series.add_prefix` and :meth:`DataFrame.add_prefix` support an ``axis`` argument. If ``axis`` is set, the default behaviour of which axis to consider can be overwritten (:issue:`47819`)
 - :func:`assert_frame_equal` now shows the first element where the DataFrames differ, analogously to ``pytest``'s output (:issue:`47910`)
 - Added new argument ``use_nullable_dtypes`` to :func:`read_csv` to enable automatic conversion to nullable dtypes (:issue:`36712`)
+- Added new global configuration, ``io.nullable_backend`` to allow ``use_nullable_dtypes=True`` to return pyarrow-backed dtypes when set to ``"pyarrow"`` in :func:`read_parquet` (:issue:`48957`)
```

Comment:
Should engine default to arrow if the option Is set to arrow?

Reply:
I'd be a little hesitant to override a user specifying `pd.read_*(..., engine="not-arrow")` and this option forcing arrow to be used instead

Reply:
Agreed, but we could make engine=NoDefault and only override if not given. 


But I agree that we should consider this carefully